### PR TITLE
Added genesis, persistent peers for the testnet

### PR DESCRIPTION
**Pull request by askolesov**

### CODE REVIEW

**Review by ashcherbakov — APPROVED** [review]

**Review by ashcherbakov — APPROVED** [review]

File: docs/running-node.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ and contains the genesis and persistent_peers files.
 
 2. Configure dclcli:
     * `dclcli config chain-id <chain-id>`
-      * Use `dcl-test-net-chain` if you want to connect to the persistent Test Net
+      * Use `testnet` if you want to connect to the persistent Test Net
```
Comment (ashcherbakov) [inline docs/running-node.md:53]: Please change the proposed test net name in `how-to.md` (search for `dcl-test-net-chain` in the project).

File: networks/testnet/genesis.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
+{
```
Comment (ashcherbakov) [inline networks/testnet/genesis.json:1]: Previously it was assumed that the network configuration will be stored in subfolders within `deployment/persistent_chains`. Either move to this folder, or fix the docs (search for `persistent_chains` and fix in running-node.md, how-to.md, running-genesis-node.md)

File: docs/running-node.md (hunk 1 above)
Comment (askolesov) [reply docs/running-node.md:53]: Fixed

File: networks/testnet/genesis.json (hunk 2 above)
Comment (askolesov) [reply networks/testnet/genesis.json]: Moved

**Review by ashcherbakov — APPROVED** [review]

**Pull request merged**